crates/lib/tests/serde.rs

```diff
@@ -26,14 +56,16 @@ fn test_json_mappings() {
             enumm([("Hash", AlgebraicType::bytes()), ("Unit", AlgebraicType::unit())]).into(),
         ),
         ("and_peggy", AlgebraicType::option(AlgebraicType::F64)),
+        ("identity", Identity::get_type()),
     ]);
     let data = r#"
 {
     "foo": 42,
     "bar": "404040FFFF0A48656C6C6F",
     "baz": ["heyyyyyy", "hooo"],
     "quux": { "Hash": "54a3e6d2b0959deaacf102292b1cbd6fcbb8cf237f73306e27ed82c3153878aa" },
-    "and_peggy": { "some": 3.141592653589793238426 }
+    "and_peggy": { "some": 3.141592653589793238426 },
+    "identity": ["0000000000000000000000000000000000000000000000000000000000000000"]
```

Comment:
Why is this an array of strings and not just a string?

Reply:
Because the internal representation of identity is `AlgebraicType::product([("__identity_bytes", AlgebraicType::bytes())])`

Keeping the same structure here means avoids having a discrepancy between `bsatn, json, satn`.